source/scpp/src/scp/LocalNode.cpp

```diff
@@ -99,7 +99,7 @@ LocalNode::computeWeight(uint64 m, uint64 total, uint64 threshold)
 // if a validator is repeated multiple times its weight is only the
 // weight of the first occurrence
 uint64
-LocalNode::getNodeWeight(NodeID const& nodeID, SCPQuorumSet const& qset)
+LocalNode::getNodeWeight(NodeID const& nodeID, Hash const& nodeHash, SCPQuorumSet const& qset)
```

Comment:
Why ?

Reply:
I want the algorithm calculating the weight to be as it was, which uses the hash of UTXO.

Reply:
No it does not

Reply:
I'll check it again.

Reply:
The parameter you added, `nodeHash`, is not used anywhere, is it ?

Reply:
Removed the unnecessary parameter.